src/particles/PhysicsParticles.hpp

```diff
@@ -619,7 +685,7 @@ template <typename problem_t> class PhysicsParticleRegister
 #endif // AMREX_SPACEDIM == 3
 
 	// Retrieve a particle descriptor by type
-	[[nodiscard]] auto getParticleDescriptor(ParticleType type) const -> const PhysicsParticleDescriptorBase *
+	[[nodiscard]] auto getParticleDescriptor(ParticleType type) const -> PhysicsParticleDescriptorBase *
```

Comment:
We have to un-`const` this pointer in order to call `splitParticles`, since it is not a `const` function.